ios/jailed/Tweak.xm

```diff
@@ -0,0 +1,40 @@
+#import "lib/Cycript.framework/Headers/Cycript.h"
+
+%ctor {
+	CYListenServer(31337);
```

Comment:
I tried sorting the dylib files by time, but it only takes minutes into account (and it doesn't know how to ignore arch-specific dylibs)